src/Update.elm

```diff
@@ -31,6 +31,14 @@ update msg model =
         --
         -- Navigation messages
         --
+        NavigateToString url ->
+            ( model
+            , Cmd.batch
+                [ Navigation.load url
+                , updateAnalytics model (updateAnalyticsPage (pageSlug model.currentPage))
```

Comment:
Would be neat if we can track the 2 events separately so we can see if people are clicking the link or the div... so maybe add a `++ div click` ? Or something that makes sense

@roseb04 Does analytics do heatmaps by default?

Reply:
I haven't come across heatmaps, but that is not to say they aren't somewhere in there. I will have a Google